python/beeai_framework/adapters/openai/serve/openai_model.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2025 © BeeAI a Series of LF Projects, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from collections.abc import AsyncIterable, Callable
+
+from beeai_framework.adapters.openai.serve._types import OpenAIEvent
+from beeai_framework.backend import AnyMessage, ChatModel
+from beeai_framework.runnable import AnyRunnable, RunnableOutput
+from beeai_framework.utils.cloneable import Cloneable
+
+
+class OpenAIModel:
```

Comment:
This name might confuse users when importing. Right now, we have `OpenAIChatModel` and `OpenAIModel`.